bugzilla/base.py

```diff
@@ -626,8 +626,34 @@ def login(self, user=None, password=None, restrict_login=None):
         except Fault as e:
             raise BugzillaError("Login failed: %s" % str(e.faultString))
 
+    def _save_api_key(self):
+        """
+        Save the API_KEY in the config file.
+
+        If toklenfile and cookiefile are undefined, it meas that the
+        API was called with --no-cache-credentials and no change will be
+        made
+        """
+        if self.tokenfile is None and self.cookiefile is None:
+            log.info("API Key won't be updated")
+            return
```

Comment:
I assume that if there's no tokenfile and no cookiefile, the user called the API without changing in the filesystem. I'm trying to do the same here. Not sure if there's a better way.